app/src/main/java/com/android/doctorapp/ui/appointment/AppointmentViewModel.kt

```diff
@@ -331,8 +334,7 @@ class AppointmentViewModel @Inject constructor(
                         fireStore
                     )) {
                     is ApiSuccessResponse -> {
-                        setShowProgress(false)
-                        _navigationListener.value = true
+                        sendNotification("Booked appointment rejected by")
```

Comment:
Take this message in constant file because if in future need to change then only one place changed.